src/package.json

```
@@ -13,13 +14,21 @@
     "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.29.4",
     "eslint-plugin-react-hooks": "^4.5.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
```

I would prefer to use 
```suggestion
    "husky": "^6",
    "lint-staged": "^10",
```
to avoid potentially installing a major version with breaking changes.